src/stability.md

```diff
@@ -72,6 +68,22 @@ even on an `unstable` function, if that function is called from another
 Furthermore this attribute is needed to mark an intrinsic as callable from
 `rustc_const_stable` functions.
 
+## Stabilizing a library feature
+
+To stabilize a feature, follow these steps:
+
+1. Change `#[unstable]` to `#[stable(since = "version")]`.
+   `version` should be the *currently nightly*, i.e. stable+2. You can see which version is
+   currently nightly [on Forge](https://forge.rust-lang.org/#current-release-versions).
```

Comment:
I'm not a giant fan of `src/version` - it's an implementation detail and it doesn't show when the next release is. During beta week especially, it can sometimes make sense to do nightly+1 so you don't have to update the version almost immediately when the next release comes out.

Reply:
Maybe this is an argument for testing `version = ` in CI, I've seen people get it wrong by accident a lot.

Reply:
rust-lang/rust#81308 :)